File: app/src/main/java/com/artmurka/artmurkaapp/Views/Fragments/OrderStatusFragment.java
package com.artmurka.artmurkaapp.Views.Fragments;

import android.app.AlertDialog;
import android.app.Dialog;
import android.app.DialogFragment;
import android.content.DialogInterface;
import android.os.Bundle;

public class OrderStatusFragment extends DialogFragment implements DialogInterface.OnClickListener {


    @Override
    public Dialog onCreateDialog(Bundle savedInstanceState) {
        Bundle bundle = getArguments();
        String order = bundle.getString("order");
        String message = bundle.getString("message");

        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
        builder.setTitle("Заказ: " + order)
                .setMessage(message);
        builder.setPositiveButton("Добре", new DialogInterface.OnClickListener(){
            @Override
            public void onClick(DialogInterface dialog, int which) {
                dismiss();
            }
        });

        AlertDialog dialog = builder.create();

        return dialog;
    }

    @Override
    public void onClick(DialogInterface dialog, int which) {
        dismiss();
    }
}

File: app/src/main/java/com/artmurka/artmurkaapp/android/views/fragments/OrderStatusFragment.kt
package com.artmurka.artmurkaapp.android.views.fragments

import android.app.AlertDialog
import android.app.Dialog
import android.app.DialogFragment
import android.content.DialogInterface
import android.os.Bundle

import com.artmurka.artmurkaapp.R

class OrderStatusFragment : DialogFragment(), DialogInterface.OnClickListener {


    override fun onCreateDialog(savedInstanceState: Bundle): Dialog {
        val bundle = arguments
        val order = bundle.getString("order")
        val message = bundle.getString("message")

        val builder = AlertDialog.Builder(activity)
        builder.setTitle(resources.getString(R.string.fragment_order_status_order) + order!!)
                .setMessage(message)
        builder.setPositiveButton(resources.getString(R.string.fragment_order_status_ok)) { dialog, which -> dismiss() }

        val dialog = builder.create()

        return dialog
    }

    override fun onClick(dialog: DialogInterface, which: Int) {
        dismiss()
    }
}
